CONTRIBUTING.md

```diff
@@ -15,11 +32,67 @@ We welcome [issues](https://github.com/Microsoft/VoTT/issues) and [pull requests
 6. Verify linting and unit tests by running `npm test`
 7. Update docs if needed
 8. Rebase on `v2` and resolve conflicts
-9. Submit PR to `v2` branch
+9. Commit your changes using a descriptive commit message that follows [our commit message conventions](#commit-message-guidelines). Adherence to these conventions is necessary because the [change log](CHANGELOG.md) is automatically generated from these messages.
+10. Submit PR to `v2` branch
 
 Try to keep PRs small to decrease the time required to review and merge.
 
-### Style
+## Commit Message Guidelines
+We have adopted standards similar to [Angular](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit) for how our git commit messages should be formatted. We believe that this leads to more readable messages that are easier to follow when looking through the project history and in addition, are used to generate the [VoTT change log](CHANGELOG.md).
+
+
+### Commit Message Format
+Each commit message consists of a **header**, a **body** and a **footer**.
+
+```
+<type>: <short description>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+### Header
+#### Type
+Must be one of the following:
+
+* **build**: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+* **ci**: Changes to our CI configuration files and scripts (example scopes: Circle, BrowserStack, SauceLabs)
```

Comment:
should we make these example scopes specific to what we're using? we are not using gulp, broccoli, CircleCI, etc.

Reply:
ah, good point... we elected to not include scope.  I'll take this out.